lib/cfn-nag/custom_rules/EMRClusterSecurityConfigurationAttachedRule.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class EMRClusterSecurityConfigurationAttachedRule < BaseRule
+  def rule_text
+    'EMR Cluster should specify SecurityConfiguration.'
+  end
+
+  def rule_type
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W60'
+  end
+
+  def audit_impl(cfn_model)
+    violating_emr_clusters = cfn_model.resources_by_type('AWS::EMR::Cluster').select do |cluster|
+      # Warn if SecurityConfiguration property is not set...
+      next cluster unless cluster.securityConfiguration
```

Comment:
this works - one style suggestion/nit is that "next" is sort of an imperative way to look at this loop vs. a predicate.... you could rewrite this bit to  be: cluster.securityConfiguration.nil? || security_config_reference_not_found?(cluster)

Reply:
Thanks!  Updated to make this cleaner in the latest commit.